config/initializers/monkeys.rb

```diff
@@ -0,0 +1 @@
+require 'monkeys/symbol'
```

Comment:
lib/core_extensions.rb is the place to put monkey patches, but you might see in my register_rebuild suggestion that I also included `N_('TFTP')` as an example.  You might be able to keep both the method name and a user-friendly name registered.